src/index.html

```diff
@@ -2,15 +2,39 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="big-search" data-qa="big">
+      <label class="big-label">
+        <img 
+          src="./images/Search.svg" 
+          alt="icon"
+          class="big-icon">
+        <input 
+          type="text" 
+          class="big-input" 
+          data-qa="keypress" 
+          placeholder="Try “Los Angeles”">
+      </label>
+    </form>
+
+    <form class="small-search" data-qa="small">
+      <label class="small-label">
+        <img 
+          src="./images/Search.svg" 
+          alt="icon"
+          class="small-icon">
+        <input 
+          type="text" 
+          class="small-input" 
+          data-qa="keypress" 
+          placeholder="Try “Los Angeles”">
+      </label>
```

Comment:
checklist 2-nd
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md

Reply:
I can't add img in input. That's way background work only on input. What i must doing?